Kudu.Core/Helpers/PostDeploymentHelper.cs

```diff
@@ -131,7 +142,20 @@ public static bool IsAzureEnvironment()
             {
                 triggers.Add(new Dictionary<string, object> { { "type", "routingTrigger" } });
             }
-
+            // Add hubName to each Durable Functions trigger
+            if (!string.IsNullOrEmpty(hubName))
+            {
+                foreach (var trigger in triggers)
+                {
+                    if (trigger.ContainsKey("type")
+                    && (trigger["type"].ToString().Equals("orchestrationTrigger", StringComparison.OrdinalIgnoreCase)
```

Comment:
>&& [](start = 20, length = 2)

Me just being nit-picky as usual, but there are lots of whitespace inconsistencies in this change. It would be nice to have them cleaned up.